docs/developers/extension-development.md

```diff
@@ -144,18 +144,31 @@ class MyAwesomeExtension {
 
 ### static .info
 
-The `info` property for the class must contain an object with a `name` property that has a unique name for the extension.
+The `info` property for the class must contain an object with a `name` property that has a unique name for the extension, a `version` property that has the version string, and a `data` parameter that includes information about the `data` generated by the extension.
 
 ```js
+import { version } from '../package.json';
+
 class MyAwesomeExtension {
 
 }
 
 MyAwesomeExtension.info = {
-  name: 'awesome'
+  name: 'awesome',
+  version: version, // can also be hardcoded as `version: "1.0.1"`
```

Comment:
but this should only be done when not using the build tools.

Reply:
Clarified.